.circleci/config.yml

```diff
@@ -19,12 +19,12 @@ executors:
 
   builder:
     docker:
-      - image: omisegoimages/elixir-omg-builder:dev-9a75a07
+      - image: omisegoimages/elixir-omg-builder:stable-20191024
```

Comment:
Can you use a git SHA in here please? This is more deterministic so we know which code we're using to build. How about:

`stable-9a75a07-20191024`